register.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Credard - Register</title>
+  <link rel="stylesheet" href="auth.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <nav>
+    <ul>
+      <li><img src="assets/images/logo.png"></li>
+      <li style="float: right;">
```

Comment:
The button is not the one we have in the design, kindly make corrections and instead of using float use flex instead.
![image](https://user-images.githubusercontent.com/60205816/182848011-94564196-44c8-449f-999b-a41a75cd0ac8.png)